Issue 5778 (#5901)
dapr/dapr: charts/dapr/README.md

| `dapr_dashboard.replicaCount`           | Number of replicas                                                                                                                                                     | `1`                |
| `dapr_dashboard.logLevel`               | service Log level                                                                                                                                                      | `info`             |
| `dapr_dashboard.image.registry`         | docker registry                                                                                                                                                        | `docker.io/daprio` |
| `dapr_dashboard.image.imagePullSecrets` | docker images pull secrets for docker registry                                                                                                                         | `docker.io/daprio` |
| `dapr_dashboard.image.name`             | docker image name                                                                                                                                                      | `dashboard`        |

> | `dapr_dashboard.image.tag`              | docker image tag                                                                                                                                                       | `"0.6.0"`          |

Reviewer: Change default to `0.11.1`.

Author: Fixed with last commit 346b99d

| `dapr_dashboard.serviceType`            | Type of [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) to use for the Dapr Dashboard service | `ClusterIP`        |
| `dapr_dashboard.runAsNonRoot`           | Boolean value for `securityContext.runAsNonRoot`. You may have to set this to `false` when running in Minikube                                                         | `true`             |
| `dapr_dashboard.resources`              | Value of `resources` attribute. Can be used to set memory/cpu resources/limits. See the section "Resource configuration" above. Defaults to empty                      | `{}`               |
| `dapr_dashboard.ingress.enabled`        | Boolean value for enabling the creation of the ingress resource                                                                                                        | `false`            |
| `dapr_dashboard.ingress.className`      | ingress className of the ingress controller (e.g.nginx)                                                                                                                | ``                 |

> | `dapr_dashboard.ingress.host`           | Fully qualified hostname of the dashboard URL (e.g dapr.127.0.0.1.nip.io)                                                                                              | ``                 |

Reviewer 2: ```suggestion
| `dapr_dashboard.ingress.host`           | Fully qualified hostname of the dashboard URL (e.g `dashboard.dapr.local`) | ``  |
```

> | `dapr_dashboard.ingress.tls.enabled`    | Boolean value for enabling the HTTPS/TLS endpoint                                                                                                                      | `false`            |

Reviewer 2: ```suggestion
| `dapr_dashboard.ingress.tls.enabled`    | If true, enables TLS on the ingress for the Dashboard                                                                                                                      | `false`            |
```

> | `dapr_dashboard.ingress.tls.secretName` | Secret name of the kubernetes tls secret (key/certificate)                                                                                                             | ``                 |

Reviewer 2: ```suggestion
| `dapr_dashboard.ingress.tls.secretName` | Name of the Kubernetes secret containing the TLS certificate (key/certificate) for the Dashboard. Ignored if `dapr_dashboard.ingress.tls.enabled` is `false`. | ``                 |
```
